cirq-google/cirq_google/api/v2/results_test.py

```diff
@@ -20,10 +20,10 @@ def test_pack_bits(reps):
 q = cirq.GridQubit  # For brevity.
 
 
```

Comment:
I think you mentioned offline that you need to add tests, but just to be sure - please add tests cases that use the new repeated keys and not only single keys.

Reply:
Added tests.